src/core/trtserver.cc

```diff
@@ -1017,13 +1017,42 @@ TRTSERVER_UnloadModel(TRTSERVER_Server* server, const char* model_name)
   ni::InferenceServer* lserver = reinterpret_cast<ni::InferenceServer*>(server);
 
   ni::ServerStatTimerScoped timer(
-      lserver->StatusManager(), ni::ServerStatTimerScoped::Kind::CONTROL);
+      lserver->StatusManager(), ni::ServerStatTimerScoped::Kind::MODELCONTROL);
 
   RETURN_IF_STATUS_ERROR(lserver->UnloadModel(std::string(model_name)));
 
   return nullptr;  // success
 }
 
+TRTSERVER_Error*
+TRTSERVER_RegisterSharedMemory(
```

Comment:
This should be TRTSERVER_ServerRegisterSharedMemory to follow the naming convention. The model load/unload is also wrong, I'll ask @GuanLuo to fix that.